### PR TITLE
New design for our homepage

### PR DESCRIPTION
**Pull request by zner0L**
This is not done, yet. Currently missing:

- [x] Working request type buttons
- [x] Request reminder notification widget
- [ ] A "Contribute" section (do we want this?)

I would love some feedback on the current design.

### CODE REVIEW

File: src/i18n/de.json [hunk 1]
```diff
@@ -63,6 +54,18 @@
         "home-feature-your-rights-portabillity-title": "Recht auf Datenübertragbarkeit",
         "home-feature-your-rights-portabillity": "Die Daten, die Du einen Unternehmen zu Dir mitgeteilt hast, gehören Dir. Du hast ein Recht, diese Daten von dem Unternehmen in einem gängigen Format zu erhalten, sodass Du sie zu einem anderen Unternehmen übertragen kannst.",
         "home-read-more-url": "/blog/deine-dsgvo-rechte",
+        "home-claim-title": "So kommst Du an Deine Daten:",
+        "home-claim-choose-company-title": "Wähle Unternehmen aus",
+        "home-claim-choose-company": "Such Dir Unternehmen aus unserer Datenbank aus, oder ergänze ein eigenes. Du weiß nicht wen Du Anfragen sollst? Dann haben wir auch ein paar Vorschläge.",
```
Comment (baltpeter) [inline src/i18n/de.json:59]:
```suggestion
        "home-claim-choose-company": "Such Dir Unternehmen aus unserer Datenbank aus, oder ergänze eigene. Du weiß nicht, wen Du anfragen sollst? Dann haben wir auch ein paar Vorschläge.",
```

File: src/i18n/de.json [hunk 2]
```diff
@@ -63,6 +54,18 @@
...
+        "home-claim-send-title": "Verschicke die Anfragen",
+        "home-claim-send": "Aus Deinen Angaben generieren wir Deine Anfragen und Du verschickst sie selbst, meist per E-Mail. Wir kriegen Deine Daten dabei nie zu sehen.",
+        "home-claim-react-title": "Erhalte Antworten & reagiere",
+        "home-claim-react": "Im besten Fall bekommst du <strong>nach einem Monat</strong>, was du angefragt hast. Schau regelmäßig vorbei und wir erinnern Dich, falls es Zeit wird beim Unternehmen nachzuhaken. Wir helfen dir auch, die Antworten einzuschätzen und dich ggf. zu beschweren.",
```
Comment (baltpeter) [inline src/i18n/de.json:63]:
```suggestion
        "home-claim-react": "Im besten Fall bekommst du <em>innerhalb eines Monats</em>, was du angefragt hast. Schau regelmäßig vorbei und wir erinnern Dich, falls es Zeit wird beim Unternehmen nachzuhaken. Wir helfen dir auch, die Antworten einzuschätzen und dich ggf. zu beschweren.",
```

**Review by baltpeter — DISMISSED** [review]

**Comment by mal-tee** [comment]
Just chiming in with some points, I really like the current draft!

- I get a lot of JS errors on all subpages on this branch, but I guess that is already known?
- Can we add a little non-textual hint that we are talking about data **other companies** store about the user?
image
With these buttons and headlines users could first think that this is only about data we have on them? I think we need another visual clue?
- I am not really a fan of the small margins in the 1-2-3-Box :D
- Do we intend to re-add the background-wave? Atm. the first 3/4 of the screen are very empty and vast.

**Comment by zner0L** [comment]
Note to self: Apparently the reaction links are not reachable via the keyboard.

File: scripts/webpack-i18n-loader.js [hunk 3]
```diff
@@ -32,6 +32,19 @@ module.exports = function (content) {
...
+
+        // To avoid translating the same strings multiple times, we can import other translations into hugo here
+        const import_keys = [
+            ['generator', 'access-request-statement'],
+            ['generator', 'erasure-request-statement'],
+            ['generator', 'rectification-request-statement'],
+            ['generator', 'objection-request-statement'],
+        ];
+        import_keys.forEach((key) => {
+            const property = key.reduce((obj, k) => obj?.[k], data);
```
[the remark this answers is not in the document]
Comment (zner0L) [reply scripts/webpack-i18n-loader.js:44]: I forgot for a moment that I know exactly how `data` looks like. Thanks for pointing out my stupidity!

**Comment by zner0L** [comment]
Why did you break the translation in the rebase?

**Review by baltpeter — CHANGES_REQUESTED** [review]
Great, thanks. I'll forgive the review comments you ignored, even though I still disagree. :P

**Review by baltpeter — APPROVED** [review]
Oops, too many buttons.

**Pull request merged**